.github/workflows/sanity-checks.yaml

```
@@ -6,10 +6,10 @@ on:
     tags-ignore:
       - '**'
     branches:
-      - master
+      - main
```

Run both on main and master:
```suggestion
      - main
      - master
```